src/main/kotlin/com/finn/carrental/persistence/UserRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.finn.carrental.persistence
+
+import com.finn.carrental.persistence.entities.UserEntity
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.MongoRepository
+
+interface UserRepository : MongoRepository<UserEntity, String> {
+    fun findOneById(id: ObjectId): UserEntity
```

Comment:
I believe the return type  should be nullable (e.g. user not found)
```suggestion
    fun findOneById(id: ObjectId): UserEntity?
```